package/mash-client.spec

```diff
@@ -46,12 +51,13 @@ with a MASH server instance.
 %setup -q
 
 %build
-python3 setup.py build
+%pyproject_wheel
 mkdir -p man/man1
-python3 setup.py --command-packages=click_man.commands man_pages --target man/man1
+python3.11 setup.py --command-packages=click_man.commands man_pages --target man/man1
```

Comment:
This will when `%{?sle15_python_module_pythons}` changes it's target or when building for multiple interpreters. Should be `%{python_flavor}`